content/pandas/concepts/dataframe/terms/drop/drop.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: '.drop()'
+Description: 'Returns a DataFrame object with rows or columns removed based on column or index names.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Data Structures'
+  - 'Pandas'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.drop()` method returns a new [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) object with rows or columns removed based on column or index names. The original `DataFrame` object, used to call the method, remains unchanged.
+
+## Syntax
+
+```pseudo
+# Drop names from specified axis.
+df = DataFrame.drop(names, axis)
+
+# Drop names from columns.
+df = DataFrame.drop(columns=names)
+
+# Drop names from rows. (axis defaults to "0")
+df = DataFrame.drop(names)
+```
+
+- `names` is a single label or a list of the items to drop.
+- `axis` is equal to `0` for dropping rows and `1` for dropping columns, it defaults to `0`.
+- `columns` or `index` can be used to specify the labels to drop without using `axis`.
+
+`DataFrame.drop()` has the following parameters:
+
+| Parameter Name | Data Type                | Usage                                                                                                                             |
+| -------------- | ------------------------ | --------------------------------------------------------------------------------------------------------------------------------- |
+| `labels`       | single label or list     | The labels to drop from columns or index as specified by `axis`.                                                                  |
+| `axis`         | 0/1 or 'index'/'columns' | Specifies if `labels` refers to columns or indices. Defaults to `0`.                                                              |
+| `index`        | single label or list     | Specifies index labels to drop. Ignores `axis`.                                                                                   |
+| `columns`      | single label or list     | Specifies column labels to drop. Ignores `axis`.                                                                                  |
+| `level`        | index of level name      | For MultiIndex dataframes, the level from which the label(s) will be removed.                                                     |
+| `inplace`      | bool                     | If `True`, alters the existing `DataFrame` rather than returning a new one. Defaults to `False`.                                  |
+| `errors`       | 'ignore'/'raise'         | Specifies if exception is raised for non-existent labels ('raise') or if those labels are ignored ('ignore'). Default is 'raise'. |
+
+## Example
+
+```py
```

Comment:
Again, I think the output should be separated out into its own snippet away from the actual Python code (with `>>>`s removed).